src/App.tsx

```diff
@@ -1,23 +1,42 @@
-import React, {useEffect} from 'react';
+import React, { useEffect, useState } from 'react';
 import { getWeatherByLatLon } from "./weatherApi";
+import { googleMapsApi } from "./apis/googleMapsApi"
 import logo from './logo.svg';
 import './App.css';
 
 function App() {
-
+  const [searchString, setSearchString] = useState<string>("");
   useEffect(() => {
+
     getWeatherByLatLon(19.4150109, -98.1410678)
-        .then(res => console.log(res))
-        .catch(err => console.log(err));
+      .then(res => console.log(res))
+      .catch(err => console.log(err));
   }, [])
 
+
+  function onChangeEntry(dataString: string) {
+    setSearchString(dataString)
+  }
+
+  function submitSearch() {
+    getLatLon(searchString);
```

Comment:
Why are you calling `getLatLon` instead of call `googleMapsApi(searchString)`. I think it's a little redundant.

**Also we should use arrow functions**

Reply:
You are right it is very redundant thanks for the comment